source/project-stories/petition-map.erb

```diff
@@ -0,0 +1,152 @@
+
+quote_3:
+  quote: "Since the EU Referendum petition, by popular demand the map has been updated with a new colouring option – weighting each constituency with its population. Hovering over each constituency also shows you the percentage of constituents that have signed. These new features make Petition Map an even more useful tool for examining regional engagement with politics."
+  source: "Cale Tilford, Developer, Unboxed"
+
+team:
+  title: "Unboxed team"
+  people:
+  - full_name: "Cale Tilford"
+    image: "/project-stories/petition-map/team/cale.png"
+    link: "/people#cale-tilford"
+
+site:
+  link: "http://petitionmap.unboxedconsulting.com/"
+  text: "Visit the Petitions Map"
+
+links:
+  - title: "Public sector collaboration with GDS to relaunch Petitions platform for UK citizens"
+    source: ""
+    type: "project story"
+    link: "/project-stories/petitions/"
+
+  - title: "Record-breaking EU 2nd referendum petition hits 3.9 million signatures and rising"
+    source: ""
+    type: "news"
+    link: "/news/record-breaking-eu-2nd-referendum-petition-hits-3-9-million-signatures-and-rising-most-popular-parliament-petition-ever/"
+
+---
+
+<%= partial 'project-stories/page_header' %>
+
+<div class="container" data-header-waypoint>
+  <%= partial 'project-stories/full_image', locals: { section: current_page.data.header } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.introduction } %>
+
+  <section class="highlight--petition-map">
+    <h2 class="highlight__title">
+      <%= current_page.data.statistics_1.title %>
+    </h2>
+
+    <div class="highlight__tile--two">
+      <p class="highlight__large-text"><%= current_page.data.statistics_1.number %></p>
+      <p class="highlight__small-text"><%= current_page.data.statistics_1.text %></p>
+      <p class="highlight__large-text"><%= current_page.data.statistics_2.number %></p>
+      <p class="highlight__wide-text"><%= current_page.data.statistics_2.text %></p>
+    </div>
+  </section>
+  
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.about } %>
+
+	<blockquote class="quote">
+	  <aside class="quote__right-section">
+	    <p class="quote__text--centered">
+	      <%= current_page.data.quote_1.quote %>
+	    </p>
+	    <p class="quote__source">
+	      <%= current_page.data.quote_1.source %>
+	    </p>
+	  </aside>
+	  <img class="quote__source-image" src="<%= current_page.data.quote_1.image %>" alt="<%= current_page.data.quote_1.source %>"/>
+	</blockquote>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.open_source } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.quote_2 } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.public_following } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.quote_3 } %>
+
+  <%= partial 'project-stories/project_team_single' %>
```

Comment:
BEM stands for block__element--modifier, so the `project-team__headshot-container` and `project-team__container--X` classes are slightly disconnected (perhaps we could tidy that up?)

Otherwise we have our main block:
- `.project-team`

then our elements:
- `.project-team__title`
- `.project-team__headshot-container`

and lastly, modifiers which are somewhat flexible in that they could modify blocks or elements, in our case it's modifying the `headshot-container` element:
- `.project-team__container--1`
- ...

[Harry Robert's article on BEM](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) (but also generally his whole blog) is a great resource to refer to.